### PR TITLE
Kk aws textract

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/appstore/connectors/aws/aws-textract.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+3.2.3 EntityTypes
+
+| Name | Caption |
+| --- | --- |
+| `KEY` | KEY |
+| `VALUE` | VALUE |
+| `COLUMN_HEADER` | COLUMN_HEADER |
+
+3.2.4 AWS_Region
+
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws/aws-textract.md:156]: Can we add a line here like: For supported Regions check: https://docs.aws.amazon.com/general/latest/gr/textract.html

File: content/en/docs/appstore/connectors/aws/aws-textract.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    1. Double-click the activity. 
+    2. Select the **AnalyzeDocumentRequest** entity. 
+    3. Add the members **GetForms** and **GetTables**, set the value to **true**, and then click **OK**.
+    The configured activity properties look like this: 
+    {{< figure src="/attachments/appstore/connectors/aws-textract/create-object-properties.png" >}}
+19. In the **Toolbox** pane, in the **AmazonTextractConnector** section, find the **AnalyzeDocument** activity.
+20. Drag the **AnalyzeDocument** activity onto the work area of your microflow between the **Create object** activity and the microflow end event.
+21. Configure the **AnalyzeDocument** activity by doing the following steps:
+    1. Double-click the activity.
+    2. Set the values for **AWS_Region**, **AnalyzeDocumentRequest**, and the **FileDocument** parameter.
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws/aws-textract.md]: Can we add, for AWS_Region, a link mentioning: Supported Regions (https://docs.aws.amazon.com/general/latest/gr/textract.html)

File: content/en/docs/appstore/connectors/aws/aws-textract.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-textract.md:156]: Done.

File: content/en/docs/appstore/connectors/aws/aws-textract.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws/aws-textract.md]: Done.

**Pull request merged**